effect/shared/src/main/scala/com/monovore/decline/effect/CommandIOApp.scala

```diff
@@ -0,0 +1,42 @@
+package com.monovore.decline.effect
+
+import cats.effect.{IO, IOApp, ExitCode}
+import cats.implicits._
+
+import com.monovore.decline._
+
+abstract class CommandIOApp(name: String, header: String, helpFlag: Boolean = true, version: String = "") extends IOApp {
+
+  def commandOpts: Opts[IO[ExitCode]]
```

Comment:
I'd be tempted to name this just `opts` or `main`.

Reply:
`main` it's ok for me